packages/@okta/vuepress-site/code/oie/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Identity Engine SDKs & Samples
+---
+
+# Identity Engine SDKs & Samples
+
+<ApiLifecycle access="ie" />
+
+## What's new
+
+[Identity Engine](/docs/concepts/ie-intro/) introduces new identity management capabilities with greater flexibility and customization. To take advantage of these new features and for a better development experience, use the Identity Experience (IDX) SDKs to manage authentication in your apps. These new SDKs are recommended for orgs using the Identity Engine.
+<!-- Identity Engine introduces new capabilities and design patterns. To take advantage of these new features and provide a better development experience, new Okta Identity Experience (IDX) SDKs are available and recommended for orgs using Identity Engine. -->
+
+Experiment with sample apps that showcase Identity Engine using the following [authentication approaches](/docs/concepts/redirect-vs-embedded/):
+<!-- We also have new sample apps for you to experiment with Identity Engine authentication, for multiple authentication approaches: -->
+
+* **Redirect**: These sample apps demonstrate how to redirect users to an Okta-hosted sign-in page, and then receive users redirected back from Okta after sign in. This approach is recommended for most developers, as it is easier to build and maintain.
```

Comment:
In the Best Practices for InfoDev Styles section of the style guide, we talk about not using sign-in as a noun. 

```suggestion
* **Redirect**: These sample apps demonstrate how to redirect users to an Okta-hosted sign-in page, and then receive users redirected back from Okta after users sign in. This approach is recommended for most developers, as it is easier to build and maintain.
```